tests/validator/validator.cpp

```diff
@@ -2132,3 +2132,76 @@ TEST(Validator, interfaceTestingNotValid)
     EXPECT_EQ(expectedErrors.size(), validator.errorCount());
     checkExpectedErrors(expectedErrors, validator);
 }
+
+/* 
+// TODO Commenting out for now, but still demonstrates a bug in the way we store equiv variables
+TEST(Validator, fixUpEquivVariables)
```

Comment:
Again I think this test should be in it shows behaviour of the library it looks at usage in a different way all of which is important in understanding what the library does and does not do.

Reply:
yes, but perhaps not in this branch?

Reply:
I would change the comments to reflect how we are thinking about what is going on though not what alternative way things should be happening. And therefore changing the answers to what the library is actually giving.

Reply:
Done